### ISSUE TITLE
Add multiprocessing for fixed batch runs

### ISSUE THREAD

**Issue by jdfoote** [projectmesa/mesa#1092]
**What's the problem this feature will solve?**

Currently, multiprocessing is supported for batch runs that do parameter sweeps (i.e., the BatchRunner class) but not for the FixedBatchRunner class (which I believe is the parent class).

**Describe the solution you'd like**

I would love to have multiprocessing support added for FixedBatchRunner.

**Comment by tpike3** [comment]
@jdfoote We recently just merged a new batch_runner if you install Mesa from git ```pip install https://github.com/projectmesa/mesa```

and then use  the new command [batch_run](https://github.com/projectmesa/mesa/blob/cda0495a4c7ca9656cbefd393d9a085b7cbf3367/mesa/batchrunner.py#L34) you should be able to do fixed parameters. If you follow the link it will take you to the function with the documentation.  

Let us know how it goes.

**Comment by jdfoote** [comment]
Correct me if I'm missing something, but it looks like this only supports parameter sweeps, with runs for all combinations of all iterable parameters?

What I'd like to do is send a set of list of parameters for each set of model runs. Something like:

```{python}
[{'param1': 'x1', 'param2': 'x2', ...}, {'param1': 'y1', 'param2': 'y2'}, ...]
```

**Comment by tpike3** [comment]
Fixed with 0.8.9 release. Please let me know if you have any issues.
